pvlib/test/test_irradiance.py

```diff
@@ -260,8 +260,7 @@ def test_globalinplane():
         40, 180, irrad_data['dhi'], irrad_data['dni'], dni_et,
         ephem_data['apparent_zenith'], ephem_data['azimuth'], airmass)
     irradiance.globalinplane(
-        aoi=aoi, dni=irrad_data['dni'], poa_sky_diffuse=diff_perez,
-        poa_ground_diffuse=gr_sand)
+        aoi, irrad_data['dni'], diff_perez, gr_sand)
```

Comment:
Does this test essentially check that `poa_components()` executes without exception?

Reply:
Correct. I only changed this test because I changed the ``globalinplane`` function signature to ``*args`` so that it could be a direct pass through to ``poa_components``. No one ever bothered to add numeric assertions to this test.

Reply:
Is there an issue for test improvement that we could put in a comment here?